babel.config.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
         },
         useBuiltIns: 'entry',
         corejs: { version: 3, proposals: false },
+        exclude: ['transform-regenerator'],
```

Comment:
This removes polyfills for JavaScript [generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Generator), [async](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) functions and the [await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await) operator. The version of nodejs we have for running tests support these natively now. The same goes for all the browsers we support. cc: @benhalpern just checking if it's OK to enable this for the tests and application. If not, I can set it to just the `test` environment in the Babel configuration file.

Reply:
Thanks for the explanation! I was just about to ask what is being excluded here and why. 😂

Reply:
To add a bit more context to this @benhalpern, we currently are not using async/await in the application portion of the codebase, so this just opens the door for people to use it without polyfills (smaller bundles).

Reply:
@nickytonline so excited about this! I think it deservers a heads up to the othe devs, we can finally use `async/await` :D